src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const generalCSQuiz = [
+  {
+    Question: "In context of Data Structures, what is a 'B-Tree'?",
+    Answer: "A data structure that represents a balanced k-ary tree",
+    Distractor1: "A data structure used to store and retrieve binary data",
+    Distractor2: "A tree structure commonly used in hierarchical databases",
+    Distractor3: "A sorting algorithm based on the divide and conquer strategy",
+    Explanation:
+      "B-tree is a data structure that represents a balanced k-ary tree",
+    Link: "https://www.google.com/amp/s/www.geeksforgeeks.org/introduction-of-b-tree-2/amp/"
```

Comment:
we only accept freeCodeCamp resources, or wikipedia

```suggestion
    Link: "https://www.freecodecamp.org/news/b-tree-self-balancing-search-index-data-structures-explained/"
```